docs/man/rpmkeys.8.md

```diff
@@ -41,29 +41,25 @@ armored public key can be added to the **rpm** database using
 ring management is performed exactly like package management. For
 example, all currently imported public keys can be displayed by:
 
-**rpm -q gpg-pubkey**
-
-A more convenient way to display them is
-
 **rpmkeys** **\--list**
 
-More details about a specific public key, when imported, can be displayed by
-querying. Here\'s information about the Red Hat GPG/DSA key:
-
-**rpm -qi gpg-pubkey-db42a60e**
+This lists all currently imported public keys (aka certificates)
+by their fingerprint and User ID. The fingerprint is the handle
```

Comment:
s/User ID/user ID/.  (In case you picked this up from our documentation, I'm sorry!  It's also not consistently correct there.)

Reply:
I must've been thinking some string I saw in rpm codebase :sweat_smile:  Yup, uppercase makes no sense there.